soc/bflb/bl60x/Kconfig.defconfig

```diff
@@ -5,9 +5,9 @@
 if SOC_SERIES_BL60X
 
 # On BL602 and BL604, not all values work, here is a list of some that work:
-# 5000, 1000, 100, 10, 1
+# 1000, 100, 10, 1
 config SYS_CLOCK_TICKS_PER_SEC
-	default 5000
+	default 1000
```

Comment:
Ideally this could came from devicetree. Then user can select the Tick there.

Reply:
How would one do that? I havent seen it done from DT yet, another property like the timebase one?

Reply:
We use the `riscv,machine-timer` to tick the system but that bind does not have the TICKS_PER_SEC int property. We could add this there. Then you can use the devicetree Kconfig extensions to pick the value and set the CONFIG_SYS_CLOCK_TICKS_PER_SEC.

I would add a help entry to tell what are the valid numbers and drop the comments.

Reply:
That's not related to the machine timer here, this is a zephyr setting (that applies to all platforms) for the kernel ticks.

Reply:
So, why do we use machine timer ?

Reply:
For the counting of the time on riscv platforms? eg this is a kernel setting vs the other being a peripheral/hardware property/setting

Reply:
It seems like other SoCs could also benefit from it.
Would it be worth splitting this suggested change into a different PR with goal to reduce fragmentation?

https://github.com/search?q=repo%3Azephyrproject-rtos%2Fzephyr+config+SYS_CLOCK_TICKS_PER_SEC&type=code

I remember I tried to use the devicetree for SYS_CLOCK_TICKS_PER_SEC on another RISC-V platform but did not get it work. Maybe time for me to try that again.

Reply:
It should definitely not be in this PR.

Reply:
> It should definitely not be in this PR.

I'm not asking to be made in this PR but I'm raising the topic because I think it seems to be relevant.